client/contexts/SubscriptionContext.ts

```diff
@@ -0,0 +1,14 @@
+import { createContext, useContext } from 'react';
+
+type SubscriptionContextValue = {
+	useUserSubscription: (rid: string, fields: Mongo.Query<any>) => any;
+	useUserSubscriptionByName: (name: string, fields: Mongo.Query<any>) => any;
```

Comment:
Don't pass hooks through context. Pass subscriptions to be consumed by `useSubscription` (preferred) or reactive functions to be consumed by `useReactiveValue`.

Reply:
it will be fixed as regression :x